gui.go

```diff
@@ -634,3 +642,44 @@ func (g *Gui) execKeybindings(v *View, ev *termbox.Event) (matched bool, err err
 	}
 	return matched, nil
 }
+
```

Comment:
could you add some basic documentation to this function?

Reply:
I added the documentation :)